src/common/interpolation.c

```diff
@@ -1938,11 +1862,9 @@ int dt_interpolation_resample_cl(const struct dt_interpolation *itor, int devid,
   err = dt_opencl_enqueue_kernel_2d_with_local(devid, kernel, sizes, local);
   if(err != CL_SUCCESS) goto error;
 
-#if DEBUG_RESAMPLING_TIMING
-  ts_resampling = getts() - ts_resampling;
-  fprintf(stderr, "resampling_cl %p plan:%" PRId64 "us resampling:%" PRId64 "us\n", (void *)dev_in, ts_plan,
-          ts_resampling);
-#endif
+  dt_show_times_f(&start, "[dt_interpolation_resample_cl]",
```

Comment:
Same double-counting of time as above due to loss of reading the new start time at (original) line 1866.